src/range.jl

```diff
@@ -20,7 +20,10 @@ Base.lastindex(fr::FixedRange) = length(fr)
 replace_end(::FixedEnd{N}, ::Type{SEQ}) where {N, SEQ} = type_length(SEQ) - N
 replace_end(n::Int, ::Type) = n
 
-@generated Base.getindex(seq, ::FixedRange{A, B}) where {A, B} =
+Base.getindex(seq::NTuple{N, Any}, fr::FixedRange{A, B}) where {N, A, B} = _getindex(seq, fr)
```

Comment:
Why don't you type it here too? The result will be an `NTuple`.
```suggestion
Base.getindex(seq::NTuple{N, T}, fr::FixedRange{A, B}) where {N, A, B, T} = _getindex(seq, fr)::NTuple{B-A+1, T}
```

Reply:
Thanks; done in f2407af